docker/nightly/centos7/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:latest
+
+LABEL maintainer="Jumping Yang <yzpcreate@qq.com>"
+LABEL readme.md="https://github.com/Microsoft/omi/blob/master/docker/README.md"
+LABEL description="This Dockerfile will install the latest code of OMI."
+
+ARG fork=Microsoft
+ARG branch=master
+
+SHELL ["/bin/bash", "-c"] 
+RUN yum update -y; \
+    yum install git bind-utils gcc-c++ rpm-devel pam-devel openssl-devel rpm-build krb5-devel redhat-lsb-core;
+RUN git clone https://github.com/$env:fork/omi.git -b $env:branch; \
+    cd omi; \
+    pushd Unix; \
+    ./configure --enable-system-build --enable-native-kits; \
```

Comment:
OMI, I believe, is published in the Microsoft repository. Couldn't you just grab it from there?